addons/network/XEH_preInit.sqf

```diff
@@ -0,0 +1,35 @@
+#include "script_component.hpp"
+
+ADDON = false;
+
+#include "initSettings.sqf"
+
+// Restore loadouts lost by the naked unit bug
+[QGVAR(validateLoadout), {
+    params ["_unit", "_loadout"];
+
+    if !(getUnitLoadout _unit isEqualTo _loadout) then {
+        _unit setUnitLoadout _loadout;
+    };
+}] call CBA_fnc_addEventHandler;
```

Comment:
Event should do nothing if this stuff is not enabled, for security reasons.

Reply:
What kind of argument is that even? Anyone that can call a CBA EH could do a lot more harm than calling this one...

Reply:
Except that you can use EG setUnitLoadout on the server without using the command on client.

Reply:
> I also wrote that I don't want this to be a setting. It should be enabled by config / mission config / preInit global flag really.

Why not? What is so good about denying anyone this workaround that isn't a mod or mission maker?